index.coffee

```diff
@@ -133,6 +119,14 @@ calculatePercentage = (completed, total) ->
 		percentage = Math.min(100, (100 * completed) // total)
 	return percentage
 
+prettyPercentage = (percentage) ->
+	if percentage?
+		" - #{percentage}%"
+	else
+		''
+
+PROGRESS_BAR_WIDTH = 50
```

Comment:
It'd be nice to have this as `DEFAULT_PROGRESS_BAR_WIDTH` - I thought this was a permanently fixed size rather than just a default when I first saw it